### DISCUSSION TITLE
Dev: Read all entries and update profile repeatedly after each other.

### DISCUSSION THREAD

**Discussion by real-yfprojects** [borgbase/vorta#1178] [Ideas]
Why are lines 86-90 run for each radio button instead of once?

https://github.com/borgbase/vorta/blob/4d399fe2ab726e840f0e64612c7dad6cb401b0e3/src/vorta/views/schedule_tab.py#L80-L93

**Comment by real-yfprojects** [comment]
ping @m3nu

**Reply by m3nu** [reply]
It only runs for the _selected_ radio button because of `if obj.isChecked():`

We need to go over all buttons, since we don't know which one is selected.
